tests/trilinos/trilinos_sparsity_pattern_03.cc

```diff
@@ -39,7 +39,7 @@ void test ()
   columns.add_range(3*myid, 3*myid+3);
   columns.compress();
 
-  sp.reinit(rows, columns, MPI_COMM_WORLD, 0);
+  sp.reinit(rows, columns, MPI_COMM_WORLD, (dealii::types::global_dof_index)0);
```

Comment:
On 09/02/2015 07:45 AM, Timo Heister wrote:
> the problem is that the non-template overload is only selected if it is an
> exact match and 0 is a signed int by default, right?

Yes.


 > We could introduce
> another overload with |int n_entries_per_row|. Alternatively, we could
> deprecate that reinit altogether I guess. I don't see a need for construction
> a Trilinos SP without handing it a dynamic sparsity pattern.

I guess that's probably true.